### PR TITLE
Hide toolbar when server is disconnected

### PR DESCRIPTION
**Pull request by revathskumar**
Card URL : https://trello.com/c/hMHlBGAI

### CODE REVIEW

File: app/components/layout.js [hunk 1]
```diff
@@ -42,7 +42,11 @@ class Layout extends React.Component {
         <Navbar {...this.props} handleConnect={this.connect} handleDisconnect={this.disconnect}/>
         <div className="container">
           <div className="row tools-wrapper">
-            <Toolbelt {...this.props} handshake={this.handshake} handleCpuSampling={this.cpuProfiling}/>
+            {
```
Comment (kgrz) [inline app/components/layout.js]: Can this be moved to a function that has a normal `if else`? I'm not too sure about a multiline ternary operator

Comment (kgrz) [inline app/components/layout.js]: Works otherwise

Comment (revathskumar) [inline app/components/layout.js]: Should I move to a method or variable?

Comment (revathskumar) [inline app/components/layout.js]: @kgrz updated the PR. used variable.

File: app/components/layout.js [hunk 2]
```diff
@@ -37,12 +37,18 @@ class Layout extends React.Component {
   }
 
   render() {
+    let toolbelt = null;
+    if (this.props.status === 'CONNECTED') {
```
Comment (kgrz) [inline app/components/layout.js:41]: This should be a function, I think I was a bit ambiguous. My idea is, if this is put inside a function, the will be no need to set the `toolbelt` value to `null`

``` javascript
const renderToolbelt = (props, handshake, cpuProfiling) => {
  if (props.status === 'CONNECTED') {
    return <Toolbelt {...props} handshake={handshake} handleCpuSampling={cpuProfiling} />;
  } else {
    return null;
}
```

But I'm going to merge this now.

Comment (revathskumar) [inline app/components/layout.js:41]: @kgrz Noted.

**Pull request merged**